hepcrawl/extractors/iop_parser.py

```diff
@@ -106,11 +106,10 @@ def parse_node(self, response, node):
 
     def get_arxiv_eprints(self, node):
         arxiv_eprints = []
-
         arxivs_raw = node.xpath("//custom-meta-group/custom-meta/meta-value/text()")
         for arxiv in arxivs_raw:
-            ar = arxiv.extract().replace('arXiv:', '')
-            if ar:
-                arxiv_eprints.append({'value': ar})
+            arxiv_value = arxiv.extract().replace('arXiv:', '')
+            if arxiv_value:
+                arxiv_eprints.append({'value': re.sub('v[0-9]$', '', arxiv_value)})
```

Comment:
we can do a regex to catch both cases `arxiv` and `v`

Reply:
good idea

Reply:
in general is more efficient to use compile in python especially if we iterate through a list. Example https://pynative.com/python-regex-compile/